doc/tutorial/Client.lhs

```diff
@@ -107,6 +107,7 @@ data BaseUrl = BaseUrl
   { baseUrlScheme :: Scheme -- ^ URI scheme to use
   , baseUrlHost :: String   -- ^ host (eg "haskell.org")
   , baseUrlPort :: Int      -- ^ port (eg 80)
+  , baseUrlPath :: String   -- ^ path (eg "/a/b/c")
```

Comment:
Yes. One /could/ write tests that type `a` is isomorphic to `b`, but then we'd need to have a qualified annotation somewhere.